tajo-plan/src/main/java/org/apache/tajo/plan/ExprAnnotator.java

```diff
@@ -678,8 +679,26 @@ public EvalNode visitGeneralSetFunction(Context ctx, Stack<Expr> stack, GeneralS
     }
   }
 
+
   public static final Set<String> WINDOW_FUNCTIONS =
-      Sets.newHashSet("row_number", "rank", "dense_rank", "percent_rank", "cume_dist", "first_value", "lag");
+      Sets.newHashSet("row_number", "rank", "dense_rank", "percent_rank", "cume_dist", "ntile", "first_value", "last_value", "lag");
```

Comment:
It is because 'lead' function is implemented not as WindowAggFunc but as AggFunction.
I think it should be re-implemented as WindowAggFunc 
but it needs more consideration about handling of arguments and default output value
so that I think it is better to handle as separate issue.